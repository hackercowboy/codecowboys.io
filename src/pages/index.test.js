import React from 'react';
import { shallow } from 'enzyme';

import Index from './index.page';

describe('<Index/>', () => {
  it('should render', () => {
    const wrapper = shallow(<Index />);
    expect(wrapper.shallow().debug()).toMatchSnapshot();
  });
});
