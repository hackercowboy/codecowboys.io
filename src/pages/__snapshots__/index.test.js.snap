// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index/> should render 1`] = `
"<Page className=\\"presentation\\">
  <Header description=\\"I am a passionate software engineer working for a Swiss company. From time to time i work on some nice projects in my free time, which i publish on this site.\\" title={[undefined]}>
    <Navigation>
      <NavigationItem id=\\"navigation-item-apps\\" target=\\"apps\\" offset={-94}>
        Apps
      </NavigationItem>
      <NavigationItem id=\\"navigation-item-about\\" target=\\"about\\" offset={-88}>
        About
      </NavigationItem>
      <NavigationItem id=\\"navigation-item-contact\\" target=\\"contact\\" offset={-94}>
        Contact
      </NavigationItem>
    </Navigation>
  </Header>
  <Parallax bgImage=\\"/images/bg-6.webp\\" bgImageAlt=\\"Background image showing the Horseshoe Bend in Arizona\\" strength={500} bgClassName=\\"react-parallax-bgimage\\" className=\\"\\" contentClassName=\\"\\" disabled={false}>
    <ApplicatinCarousel>
      <ApplicationHero image=\\"/images/letter_carousel.webp\\" title=\\"Letter\\" link=\\"/en/letter\\" appStoreLink=\\"http://itunes.apple.com/app/letter/id498506154\\" description=\\"With the Letter app you can now write your personal letters on the iPhone, iPad and iPod Touch too, when and where you want.\\" placeholderImage={[undefined]} soon={false} />
    </ApplicatinCarousel>
    <ScrollButton target=\\"apps\\" />
  </Parallax>
  <Section id=\\"apps\\" title=\\"Apps\\" subtitle=\\"A couple of great apps for you!\\" dark={false}>
    <Application title=\\"Letter\\" teaser=\\"With the Letter app you can now write your personal letters on the iPhone, iPad and iPod Touch too, when and where you want.\\" link=\\"/en/letter\\" image=\\"/images/letter_icon.svg\\" />
  </Section>
  <Parallax bgImage=\\"/images/bg-5.webp\\" bgImageAlt=\\"Background image showing the Grand Canyon in Arizona\\" strength={500} bgClassName=\\"react-parallax-bgimage\\" className=\\"\\" contentClassName=\\"\\" disabled={false}>
    <Section id=\\"about\\" title=\\"About\\" subtitle=\\"I code great apps like a cowboy!\\" dark={true}>
      <About />
    </Section>
  </Parallax>
  <Section id=\\"contact\\" title=\\"Contact\\" subtitle=\\"Please contact me if you have any questions\\" dark={false}>
    <ContactForm />
  </Section>
  <Footer />
</Page>"
`;
